pytorch_blade/pytorch_blade/BUILD

```diff
@@ -30,19 +30,19 @@ cc_library(
     name = "torch_blade",
     linkopts = ["-Wl,-rpath,$$ORIGIN"],
     deps = [
-        "//src/common_utils:torch_blade_common",
-        "//src/compiler/backends:torch_blade_backends",
-        "//src/compiler/jit:torch_blade_jit",
-        "//src/compiler/mlir:torch_blade_mlir",
+        "//pytorch_blade/common_utils:torch_blade_common",
```

Comment:
Nit: remove the target `torch_blade_include` at line 22

Reply:
done~